source/tools/kubernetes/dask-operator.md

```diff
@@ -1 +1,381 @@
+        dask.org/cluster-name: rapids-dask-cluster
+        dask.org/component: scheduler
+      ports:
+        - name: tcp-comm
+          protocol: TCP
+          port: 8786
+          targetPort: "tcp-comm"
+        - name: http-dashboard
+          protocol: TCP
+          port: 8787
+          targetPort: "http-dashboard"
+```
+
+This example shows using a `ClusterIP` service which will not expose the Dask cluster outside of Kubernetes. If you prefer you could set this to
+`LoadBalancer` or `NodePort` to make this externally accessible.
+
+It has a `selector` that matches the scheduler pod and the same ports configured.
+
+### Accessing your Dask cluster
+
+Once you have created your `DaskCluster` resource we can use `kubectl` to check the status of all the other resources it created for us.
+
+```console
+$ kubectl get all -l dask.org/cluster-name=rapids-dask-cluster
+NAME                                                             READY   STATUS    RESTARTS   AGE
+pod/rapids-dask-cluster-default-worker-group-worker-0c202b85fd   1/1     Running   0          4m13s
+pod/rapids-dask-cluster-default-worker-group-worker-ff5d376714   1/1     Running   0          4m13s
+pod/rapids-dask-cluster-scheduler                                1/1     Running   0          4m14s
+
+NAME                                  TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
+service/rapids-dask-cluster-service   ClusterIP   10.96.223.217   <none>        8786/TCP,8787/TCP   4m13s
+```
+
+Here you can see our scheduler pod and two worker pods along with the scheduler service.
+
+If you have a Python session running within the Kubernetes cluster (like the [example one on the Kubernetes page](/platforms/kubernetes)) you should be able
+to connect a Dask distributed client directly.
+
+```python
+from dask.distributed import Client
+
+client = Client("rapids-dask-cluster-service:8786")
+```
+
+Alternatively if you are outside of the Kubernetes cluster you can change the `Service` to use [`LoadBalancer`](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) or [`NodePort`](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport) or use `kubectl` to port forward the connection locally.
+
+```console
+$ kubectl port-forward svc/rapids-dask-cluster-service 8786:8786
+Forwarding from 127.0.0.1:8786 -> 8786
+```
+
+```python
+from dask.distributed import Client
+
+client = Client("localhost:8786")
+```
+
+## Example using `KubeCluster`
+
+In additon to creating clusters via `kubectl` you can also do so from Python with {class}`dask_kubernetes.experimental.KubeCluster`. This class implements the Dask Cluster Manager interface and under the hood creates and manages the `DaskCluster` resource for you.
+
+```python
+from dask_kubernetes.experimental import KubeCluster
+
+cluster = KubeCluster(name="rapids-dask",
+                      image="rapidsai/rapidsai-core:22.06-cuda11.5-runtime-ubuntu20.04-py3.9",
```

Comment:
Might be good to explicitly state `n_workers=3` (or 2 if you want to match the `kubectl` example above). It wasn't immediately clear to me why this created 3 workers and the `kubectl` example had 2.